.travis.yml

```diff
@@ -46,6 +47,13 @@ matrix:
         - npm run coverage -- --silent
         - npm run coveralls || true
         - pkill -f ganache
+    - if: env(DEPLOY_TYPE) IS present
+      script:
+        - git clone https://$DEPLOY_TOKEN@github.com/$TRAVIS_REPO_SLUG
```

Comment:
we dont need to clone
the webhook will trigger the correct repo/branch
https://docs.travis-ci.com/user/triggering-builds/

Reply:
also no need for the following two commands, "cd","checkout"

Reply:
done